config/mimic3-generic.conf

```diff
@@ -0,0 +1,2 @@
+GenericExecuteSynth "/vx/ui/ui-functions/spd-mimic3-backend -w '$DATA' | $PLAY_COMMAND"
```

Comment:
I'm surprised this single-quoted argument gets expanded. Perhaps `speech-dispatcher` is doing the shell expansion rather than the shell itself?

I worry a little bit about arbitrary text being put into these bash variables…

Reply:
yes, I think `speech-dispatcher` is doing this.

You're totally right about the concern here... we could use some input sanitization here as a layer of defense.